.github/workflows/scan-sonar.yml

```diff
@@ -82,7 +83,9 @@ jobs:
         jans-linux-setup
         jans-cli-tui
         jans-pycloudlib
-
+    permissions:
+        contents: read
```

Comment:
I have a suggestion to remove this, since it's already defined at the top-level and thus inherited

Reply:
It would have been removed if I hadn’t added the pull-request: read permission. For visibility and readability adding it makes it easier to understand. As you can notice other workflows do not have the job level permission added as the inherited content read was sufficient